models/stitch/base/adwords_accounts.sql

```diff
@@ -1 +1,5 @@
-{{ stitch_adwords_accounts() }}
\ No newline at end of file
+{% if var('etl') == 'stitch' %}
+{{ stitch_adwords_accounts() }}
+{% elif var('etl') == 'fivetran'%}
+--Fivetran does not support Core tables
```

Comment:
instead of even listing the fivetran option, I think the default for fivetran should be to disable the core tables in the`dbt_project.yml` file. you can comment these options out since the default will be stitch